### PR TITLE
add transport in personal cabinet

### PR DESCRIPTION
**Pull request by zharan96prog**

### CODE REVIEW

File: src/app/components/transport/transport.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      insuarenceNumber: ['']
+    })
+  }
+
+  ngOnInit(): void { }
+
+  onSubmit(){
+    this.transportService.addTransport(this.transportForm.value)
+    .subscribe((data:any)=>{
+      console.log(data);
```
Comment (LizaShemetovska) [inline src/app/components/transport/transport.ts]: Delete console.log

File: src/app/models/Transport.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+export class Transport{
+    constructor(
+        public id: string,
+        public producedBy: string,
+        public model: string,
+        public vinCode: string,
+        public categoryName: string,
+        public carPlate: string,
+        public color: string,
+        public insuarenceNumber: string,
```
Comment (LizaShemetovska) [inline src/app/models/Transport.ts]: InsuaranceNumber is object

File: src/app/models/Insuarance.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export class Insuarance {
+    constructor(
+        public id: string,
```
Comment (LizaShemetovska) [inline src/app/models/Insuarance.ts]: This is object, so it doesn't id

**Review by LizaShemetovska — CHANGES_REQUESTED** [review]

**Review by pinkevmladchy — APPROVED** [review]
good work

**Review by NerevitVasyan — APPROVED** [review]

**Pull request merged**